### DISCUSSION TITLE
Struggling with Virgin Media configuration.

### DISCUSSION THREAD

**Discussion by Network-Buzzard** [up-n-atom/8311#35] [Q&A]
Hi Guys.
Ive followed the guide on how to configure the WAS-110 to replace the Virgin Media Hub5x but I am getting 
"PON PLOAM Status                                   : O2.3, Serial number state".

I have tried using the S/N and MAC address on the bottom of the 5x as instructed but I found that this S/N and MAC address differs to that which is on the 'Info' page on the 5x GUI.
Also the software version used on my 5x is 2.40.13-2210.4, but the guide says use 3.7.4-2306.5.

When following the troubleshooting guide I noticed that my TX is disabled.
According to the guide, I should be getting a reading, how do I enable it?

Any advice/clarification on the parameters to be used and ideas on where to start troubleshooting next would be greatly received.
I have tried every combination I can think of to get this up.

Thanks in Advance.

**Comment by djGrrr** [comment]
For support, please visit the [8311 Discord Server](https://discord.pon.wiki)
